bin/run

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 #
-# Ensures that Library Simplified Circulation Manager scripts don't run
+# Ensures that Palace Manager scripts don't run
```

Comment:
Is there any reason for us not to abandon this script and instead using something that already does this like [flock](https://man7.org/linux/man-pages/man1/flock.1.html)?

Reply:
Probably not worth changing it at this point, since it's used to run scripts via cron and those are going to be moved to the queue-based approach.

Reply:
Probably not worth changing it at this point, since it's used to run scripts via cron and those are going to be moved to the queue-based approach.

Reply:
I think it makes sense to keep whats here like @tdilauro mentioned, since this is going away soon. Nice to know flock exists though, I hadn't heard of it before, so I appreciate the tip.